### PR TITLE
test: misc. CI tidy-ups

### PR DESCRIPTION
**Pull request by ianvernon**
* Fix grammatical errors in function names
* Add By statements to preflight checks to provide users with feedback with what is going on while test is running.
* fix go vet errors

Signed-off by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6052)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
test-me-please

**Comment by ianvernon** [comment]
Hit CI flake: https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/7563/execution/node/94/log/?consoleFull

Microscope could not get ready.

**Comment by ianvernon** [comment]
test-me-please

File: test/helpers/kubectl.go [hunk 1]
```diff
@@ -1674,6 +1675,7 @@ func (kub *Kubectl) CiliumPreFlightCheck() error {
 // CiliumControllersPreFlightCheck validates that all controllers are not
 // failing. If any of the controllers fails will return an error.
 func (kub *Kubectl) CiliumControllersPreFlightCheck() error {
+	ginkgo.By("Checking that no controllers in Cilium have failed")
```
Comment (eloycoto) [inline test/helpers/kubectl.go:1678]: I'm ok with all this By, but  be aware that maybe are too many and people may get confused 
I I would move this ones to the BeforeAll helper function and said something at the end.

`Cilium is ready, all preflight are ok and ready to test`

**Review by eloycoto — APPROVED** [review]

**Comment by ianvernon** [comment]
NodePort failure: https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/7567/testReport/junit/k8s-1/12/K8sServicesTest_Checks_service_across_nodes_Tests_NodePort/

File: test/helpers/kubectl.go (hunk 1 above)
Comment (ianvernon) [reply test/helpers/kubectl.go:1678]: @eloycoto the reason I added them here is that the PreFlightCheck does a lot of checks that have a large timeout. Adding it into each `BeforeAll` produces more boilerplate code that we have to copy and paste into each `BeforeAll`, which we can avoid by having such output put into the functions themselves. I think this is a happy medium between too little and too much output; it provides more visibility into the state of the checks without dumping a ton of output to console. However, we can remove these checks if they end up dumping a lot of non-useful output to console.

**Comment by ianvernon** [comment]
test-me-please

**Pull request merged**